docs/sources/alerting/migrating-alerts/opt-in.md

```diff
@@ -20,9 +19,9 @@ We recommend that you backup Grafana's database. If you are using PostgreSQL as
 
 To enable Grafana alerts:
 
-1. In your custom configuration file ($WORKING_DIR/conf/custom.ini), go to the [unified alerts]({{< relref "../administration/configuration.md#unified_alerting" >}}) section.
+1. In your custom configuration file ($WORKING_DIR/conf/custom.ini), go to the [unified alerts]({{< relref "../../administration/configuration.md#unified_alerting" >}}) section.
 2. Set the `enabled` property to `true`.
```

Comment:
```suggestion
1. Set the `enabled` property to `true`.
```

If you number all the markdown list entries with 1. the markdown renderer will increment the values appropriately. I find this to be less error prone than manual numbering but you do have to be careful that all the steps are properly indented (not required in this case) so that the numbering doesn't reset.

Reply:
Fixed.